dataeng/resources/snowflake-collect-metrics.sh

```diff
@@ -5,7 +5,7 @@ set -ex
 cd $WORKSPACE/analytics-tools/snowflake
 make requirements
 
-python collect_queue_depth.py \
+python $METRIC_COLLECTION_SCRIPT \
```

Comment:
I'd say that in the industry how we might typically abstract this is by writing only one script (one entrypoint) but allowing different metrics to be selected as part of an argument, like `--metric queue_depth`.  Works either way so I'm not going to block.  I've blocked enough this week!

Reply:
Just to clarify, this would mean I have 1 .groovy script for all the metric collection jobs and then each job has its own bash script, right?

Reply:
Just to clarify after our meeting: if you refactor the analytics-tools code to provide one interface for all snowflake metric gathering, you could just have one bash file in which the only thing that is parameterized would be the argument passed to your script. Something like:
```
python gather_snowflake_metrics.py --metric $METRIC
```